### ISSUE TITLE
Set Schema

### ISSUE THREAD

**Issue by jMyles** [JustinHolmesMusic/blox-office#1]
A *show* has zero or more *sets*.

A *set* has zero or more *songs*.

A *song* has the following qualities, intrinsic to it:

* One among:
** Original
** Traditional
** Cover
** Video Game Music
** Unnamed Jam
** Unnamed talk

* Sets relate to songs with one or more of the following qualities:
** Unitary (song is simply played through)
** Segue (song begins as a transition from the previous one)
** Reprise (song is revisited after the procession of other elements of the show)
** Tease

**Comment by EhlJai** [comment]
Are we thinking these have a purchase value, gift value, attendance recognition, will there be set collections, are they trophies in cases, are they traceable, do they have physical written set list components or other physical memorabilia attached, signatures of artists and or attendees written into the set stones, is everyone who was at the set somehow written into the set stone or the total number of attendees. Could the venue also be involved in collections, clout, merch?

**Comment by jMyles** [comment]
Yep, exactly.  Presumably people will "hunt" for the qualities of a set (and its songs) that they like.

One thing has surfaced since the opening of this issue: it turns out that "original" and "cover" are not intrinsic to a song; it depends who is playing it.
